pytype/vm.py

```diff
@@ -2507,6 +2507,20 @@ def byte_SETUP_FINALLY(self, state, op):
     self.store_jump(op.target, state.push(self.convert.build_none(state.node)))
     return self.push_block(state, "finally", op, op.target)
 
+  # New python3.8+ exception handling opcodes:
+  # BEGIN_FINALLY, END_ASYNC_FOR, CALL_FINALLY, POP_FINALLY
+  def byte_BEGIN_FINALLY(self, state, op):
+    return state
```

Comment:
we need to handle the stack effects for all these opcodes, and also check for the `NULL` pushed by `BEGIN_FINALLY` in `WITH_CLEANUP_START` and `WITH_CLEANUP_FINISH`

Reply:
Let me add some `TODO`s here, handling the vm is a bit complicated here since 3.8 kinda revamped how error handling is done. Doing both the changes makes the diff really large, so I'd rather get this merged in to get the basic tests working and then tackle this separately.

Reply:
For reference: https://github.com/python/cpython/commit/520b7ae27e39d1c77ea74ccd1b184d7cb43f9dcb

Reply:
okay, sounds good